### PR TITLE
Add Clear discription of creating a database.

### PR DESCRIPTION
**Pull request by henrykironde**
When creating a new database, the tool displays a file explorer window.
It is confusing if not well stated that you do not have to select a file.
The only step needed should be clicking open.

### CODE REVIEW

**Comment by orchid00** [comment]
Hi @henrykironde do you think this is something that you can add in the Instructor guide? http://www.datacarpentry.org/sql-ecology-lesson/guide/ or where would you say would be more useful?

**Comment by orchid00** [comment]
@henrykironde   Have you tested in all OS that a new window is displayed? Also, have you had any feedback on step 1 being confusing? What do you think about: 

Start a New Database **Database -> New Database**, then **Open**
(I'll like to encourage a short step)

**Comment by henrykironde** [comment]
@orchid00, I checked once again and am using mac with the new version of Firefox. 
I still think the changes in the PR provide a better explanation for the steps needed.
If no one else is facing this problem, I will go ahead and close it.

**Comment by orchid00** [comment]
Thanks, @henrykironde I will ask feedback from others. Hello, @tpoisot @tracykteal would you please give it a go and comment on this update? I agree that it needs a bit more description but I'll rather split step 1 into at least 2 steps.

File: _episodes/00-sql-introduction.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ database.  We'll be creating this database from the three `csv` files
 we downloaded earlier.  Close the currently open database and then 
 follow these instructions: 
 
-1. Start a New Database **Database -> New Database**
+1. Start a New Database **Database -> New Database**. Give a name **Ok -> Open**. No selection needed.
```
Comment (henrykironde) [inline _episodes/00-sql-introduction.md]: @orchid00  does this look good.

1. Start a New Database 
    - **Database -> New Database**. 
    - Give a name **Ok -> Open**. Creates the database in that opened folder
2. Start the import **Database -> Import**
3. Select the `surveys.csv` file to import
4. Give the table a name that matches the file name (`surveys`), or use the default
5. If the first row has column headings, check the appropriate box
6. Make sure the delimiter and quotation options are appropriate for the CSV files.  Ensure 'Ignore trailing Separator/Delimiter' is left *unchecked*.
7. Press **OK**
8. When asked if you want to modify the table, click **OK**
9. Set the data types for each field using the suggestions in the table below (this includes fields from `plots` and `species` tables also):

Comment (orchid00) [inline _episodes/00-sql-introduction.md]: looks good!

Comment (henrykironde) [inline _episodes/00-sql-introduction.md]: Thanks @orchid00, I will update as above

**Comment by henrykironde** [comment]
@orchid00, Updated. Let me know what could be improved in case.

**Comment by orchid00** [comment]
@henrykironde thanks!

**Pull request merged**